docs/user/alerting/rule-types/es-query.asciidoc

```diff
@@ -51,41 +86,13 @@ image::images/rule-types-es-query-example-action-variable.png[Iterate over hits
 
 Use the *Test query* feature to verify that your query DSL is valid.
 
-* Valid queries are executed against the configured *index* using the configured *time window*. The number of documents that
-match the query will be displayed.
+* Valid queries are executed against the configured *index* using the configured 
+*time window*. The number of documents that match the query is displayed.
 +
 [role="screenshot"]
 image::user/alerting/images/rule-types-es-query-valid.png[Test {es} query returns number of matches when valid]
 
 * An error message is shown if the query is invalid.
 +
 [role="screenshot"]
-image::user/alerting/images/rule-types-es-query-invalid.png[Test {es} query shows error when invalid]
```

Comment:
@sophiec20 The reason I added this section to the docs was because I answered multiple SDHs and questions about how this worked using an example like this and I thought it would be nice to be able to just point to the docs instead of writing it out each time. Is there another place we might put this example that would be better than here?

Reply:
@ymao1 In this case, I'll add it back with a slightly different section title – to avoid the term "de-duplication".

Reply:
@ymao1 I updated the PR via [4bbcafe](https://github.com/elastic/kibana/pull/125104/commits/4bbcafe463e2963299af8206abe4c9cd4d78568a). I changed the section name and adjusted the terminology of the section.

Reply:
Thank you! That sounds good